spec/migration_spec.rb

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 RSpec.describe Armg, skip_create_table: true do
+  let(:table_options) do
+    if ActiveRecord.gem_version < Gem::Version.new('6.1')
+      "options: \"#{MysqlHelper::TABLE_OPTIONS}\""
+    elsif MysqlHelper::MYSQL_ENGINE == 'InnoDB'
+      'charset: "utf8mb4", collation: "utf8mb4_bin"'
+    else
+      "charset: \"utf8mb4\", collation: \"utf8mb4_bin\", options: \"ENGINE=#{MysqlHelper::MYSQL_ENGINE}\""
+    end
```

Comment:
nit: I want to avoid escaping

```suggestion
    if ActiveRecord.gem_version < Gem::Version.new('6.1')
      %(options: "#{MysqlHelper::TABLE_OPTIONS}")
    elsif MysqlHelper::MYSQL_ENGINE == 'InnoDB'
      %(charset: "utf8mb4", collation: "utf8mb4_bin")
    else
      %(charset: "utf8mb4", collation: "utf8mb4_bin", options: "ENGINE=#{MysqlHelper::MYSQL_ENGINE}")
    end
```